src/agor/tools/dev_tooling.py

```
@@ -705,165 +377,130 @@ def create_development_snapshot(self, title: str, context: str) -> bool:
+    return quick_commit_push(f"📸 Create development snapshot: {title}", "📸") # Use direct name
 
-    def test_tooling(self) -> bool:
-        """Test all development tooling functions to ensure they work properly."""
-        print("🧪 Testing AGOR Development Tooling...")
 
-        # Test timestamp functions
-        print(f"📅 Current timestamp: {self.get_current_timestamp()}")
-        print(f"📁 File timestamp: {self.get_timestamp_for_files()}")
-        print(f"⏰ Precise timestamp: {self.get_precise_timestamp()}")
-        print(f"🌐 NTP timestamp: {self.get_ntp_timestamp()}")
+def test_tooling() -> bool:
```

Ruff (0.11.9)

Redefinition of unused `test_tooling` from line 49

(F811)